src/Components/Form/FormFields/DosageFormField.tsx

```diff
@@ -0,0 +1,21 @@
+import NumericWithUnitsFormField from "./NumericWithUnitsFormField";
+import { FormFieldBaseProps } from "./Utils";
+
+type Props = FormFieldBaseProps<string> & {
+  placeholder?: string;
+  value?: string;
+  autoComplete?: string;
+  className?: string | undefined;
+  disabled?: boolean;
+  min?: string | number;
+  max?: string | number;
+};
```

Comment:
Some of the props are already defined in `FormFieldBaseProps`

```suggestion
type Props = FormFieldBaseProps<string> & {
  placeholder?: string;
  autoComplete?: string;
  min?: string | number;
  max?: string | number;
};
```